README.md

```diff
@@ -72,11 +72,12 @@ If you compile RIOT for the native cpu and include the `netdev_tap` module,
 you can specify a network interface like this: `PORT=tap0 make term`
 
 #### SETTING UP A TAP NETWORK
-There is a shellscript in `RIOT/dist/tools/tapsetup` called `tapsetup` which
+There is a shell script in `RIOT/dist/tools/tapsetup` called `tapsetup` which
 you can use to create a network of tap interfaces.
 
-*USAGE*
-To create a bridge and two (or count at your option) tap interfaces:
+_USAGE_
```

Comment:
Sorry for being blunt: but I don't see the improvement of changing `*USAGE*` to `_USAGE_` as it does not have any visual effect in markdown. The actual visual improvement is the following empty line, but on the other hand one could also omit `*USAGE*` here, too.

Reply:
It's just that it renders correctly this way in my editor... I'll revert that change np